extension/_locales/en/messages.json

```diff
@@ -0,0 +1,82 @@
+      "description": "Name of the extension"
+    },
+    "extensionDescription": {
+      "message": "Translate websites in your browser without using the cloud.",
+      "description": "Description of the extension."
+    },
+    "loadingTranslationEngine": {
+      "message": "Loading Translation Engine",
+      "description": "Tells the user the translation engine is being loaded."
+    },
+    "translationLoadedWithErrors": {
+      "message": "Automatic translation is enabled but we found errors.",
+      "description": "Tells the user the translation engine was loaded but errors were found."
+    },
+    "checksumErrorsDownloadingEngine": {
+      "message": "We found checksum errors when downloading the translation engine.",
+      "description": "Tells the user there were errors downloading the translation engine."
+    },
+    "notfoundErrorsDownloadingEngine": {
+      "message": "We could not find the translation engine to download.",
+      "description": "Tells the user the translation engine could not be found to download."
+    },
+    "nodataDownloadingEngine": {
+      "message": "We found no data when downloading the translation engine.",
+      "description": "Tells the user the translation engine had not data when downloading."
+    },
+    "translationEnabled": {
+      "message": "Automatic Translation enabled.",
+      "description": "Tells the user the automatic translation was enabled."
+    },
+    "translationEnabledNoOT": {
+      "message": "Automatic Translation enabled but translation of forms is not supported for this language.",
+      "description": "Tells the user the automatic translation was enabled but form translations is not supported."
+    },
+    "timeoutDownloadingEngine": {
+      "message": "Timeout downloading the translation engine.",
+      "description": "Tells the user the download of the translation engine timed out."
+    },
+    "errorLoadingWasm": {
+      "message": "We found an error loading the translation engine.",
+      "description": "Tells the user we found and error loading the translation engine."
+    },
+    "statsMessage": {
+      "message": "Translation enabled. Words translated-per-second: $WORDSPERSECOND$.",
+      "description": "Tells the user that translation is enabled and the current performance metric of words translated per second.",
+      "placeholders": {
+        "wordspersecond" : {
+          "content" : "$1",
+          "example" : "500"
+        }
+      }
+    },
+    "downloadProgress": {
+      "message": "Downloaded $RECEIVEDLENGTH$ of $CONTENTLENGTH$.",
+      "description": "Tells the user the download progress of the engine artifacts.",
+      "placeholders": {
+        "receivedlength" : {
+          "content" : "$1",
+          "example" : "100"
+        },
+        "contentlength" : {
+          "content" : "$2",
+          "example" : "1000"
+        }
+      }
+    },
+    "displayStatisticsMessage": {
+      "message": "Display Statistics",
```

Comment:
Is "display" a verb or something else? What are the options, or is this a checkbox?

Reply:
Display is a verb. It is an option that displays the statistics if user clicks on it. Attaching a screenshot for details.
<img width="1911" alt="Screenshot 2022-02-22 at 13 26 29" src="https://user-images.githubusercontent.com/66322306/155132251-010bc7ad-1c70-448e-953a-efd498a165ec.png">

Reply:
OK. Please change the comment to something like `Command used to show statistics (display is intended as a verb).`